include/bactria/Context.hpp

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2021 Jan Stephan
+ *
+ * Licensed under the EUPL, Version 1.2 or - as soon they will be approved by
+ * the European Commission - subsequent versions of the EUPL (the “Licence”).
+ * You may not use this work except in compliance with the Licence.
+ * You may obtain a copy of the Licence at:
+ *
+ *     http://ec.europa.eu/idabc/eupl.html
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the Licence is distributed on an “AS IS” basis, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  Licence permissions and limitations under the Licence.
+ */
+
+/**
+ * \file Context.hpp
+ * \brief bactria's Context include file.
+ *
+ * This file defines the Context interface. The user should not include this directly.
+ */
+
+#ifndef BACTRIA_CONTEXT_HPP
+#   define BACTRIA_CONTEXT_HPP
+
+#   include <bactria/detail/Activation.hpp>
+#   include <bactria/Plugin.hpp>
+
+namespace bactria
+{
+    namespace detail
+    {
+        bool bactria_activated = false;
```

Comment:
Do I see correctly that this is a non-extern, non-inline global variable in a header file? This violates the ODR (one definition rule), because each including source file will have a copy of that global.

Is this even used somewhere? If yes, move it to a cpp file and declare it `extern` here. Or use `inline` with C++17.

Reply:
Thanks for catching this, I forgot to remove that part.